dockermod_root/etc/s6-overlay/s6-rc.d/init-mod-universal-themerr-jellyfin/run.sh

```diff
@@ -0,0 +1,21 @@
+﻿#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+
+# this is mostly borrowed from
+# https://github.com/linuxserver/docker-baseimage-ubuntu/blob/8166223ec8da5012e4432776b5d6b882ffdc9c02/root/etc/s6-overlay/s6-rc.d/init-adduser/run
+
+echo 'Initializing Themerr-jellyfin, as a mod for Linuxserver.io Jellyfin container'
+
+echo '
+╔═══════════════════════════════════════════════════════════════╗
+║                                                               ║
+║          Themerr-jellyfin, a LizardByte application           ║
+║                                                               ║
+║  Docs: https://docs.lizardbyte.dev/projects/themerr-jellyfin  ║
+║      Support center: https://app.lizardbyte.dev/#Support      ║
+║          Donate: https://app.lizardbyte.dev/#Donate           ║
+║                                                               ║
+╚═══════════════════════════════════════════════════════════════╝
+'
+
+lsiown abc:abc /config/data/plugins/themerr-jellyfin
```

Comment:
I think the plug-in is capitalised and doesn't have the `-jellyfin` suffix, but I'm only on my phone at the moment.
```suggestion
lsiown abc:abc /config/data/plugins/Themerr
```

Reply:
Cool, thanks for the review! The path is correct. It's set near the end of the Dockerfile.

```dockerfile
# variables
ARG PLUGIN_NAME="themerr-jellyfin"
ARG PLUGIN_DIR="/config/data/plugins"

# add files from buildstage
# trailing slash on artifacts directory copies the contents of the directory, instead of the directory itself
COPY --link --from=buildstage /artifacts/ $PLUGIN_DIR/$PLUGIN_NAME
```